gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -72,4 +73,19 @@ public void saveMapType(int type) {
   public int getSavedMapType(int defaultType) {
     return preferences.getInt(MAP_TYPE, defaultType);
   }
+
+  /**
+   * This method will return whether the user have accepted the terms or not.
+   * @return true or false depending upon the user.
```

Comment:
This comment doesn't add any new info. You can omit `@return` if the method description already makes the meaning of the return value clear.

Reply:
https://github.com/google/ground-android/pull/845